CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+## 0.18.0 (2017-11-27)
+
+**Breaking changes**
+
+- `RESTBase` now depends on Cassandra 3.11. In case SQLite backen is used no changes us required.
+- After updating `RESTBase` new keyspaces will be created for everything, so it's effectively starting the storage from scratch. All the content stored in `RESTBase` can be rerendered, so no need to transfer the data, but old keyspaces can be deleted afterwards. New keyspace names have `_ng` suffix. 
```

Comment:
`s/keyspace names have _ng/keyspace names have the _ng/`

Also, let's limit the line lengths to 80 chars.